crates/viewer/re_ui/src/notifications.rs

```diff
@@ -450,10 +450,20 @@ fn show_notification(
                 ui.horizontal_top(|ui| {
                     ui.add(level.image(ui));
 
-                    ui.horizontal_top(|ui| {
+                    ui.vertical(|ui| {
                         ui.style_mut().wrap_mode = Some(egui::TextWrapMode::Wrap);
                         ui.set_width(270.0);
-                        ui.label(egui::RichText::new(text.clone()));
+                        let (split_text, details) = text.split_once("<details>").unzip();
+                        let text = split_text.unwrap_or(text);
+                        ui.label(egui::RichText::new(text));
+
+                        if let Some(details) = details {
+                            if mode == DisplayMode::Panel {
+                                ui.collapsing_header("Details", false, |ui| ui.label(details));
+                            } else {
+                                ui.weak("Open notification toast for more details");
```

Comment:
Hmm but technically the "toast" is the temporary thing, not the card in the "permanent" menu.

```suggestion
                                ui.weak("Open notification menu for more details");
```

maybe better?

Anyways, as discussed on slack it would be cool to make the collapsing panel work consistently imho